### ISSUE TITLE
Why do i maintain a 10% accuracy rate when training this model?

### ISSUE THREAD

**Issue by NiuRc** [ShunLu91/Single-Path-One-Shot-NAS#1]
I replace the dataset with Cifar10  ,and reduce the number of layers.When i training this model ,the accuracy rate maintain 10%.It can't change

**Comment by ShunLu91** [comment]
> I replace the dataset with Cifar10 ,and reduce the number of layers.When i training this model ,the accuracy rate maintain 10%.It can't change
As far as I am concerned, I think may be the learning rate is to blame which is different from the official version and may be not appropriate. Additionally, the initial state is very important. Maybe you can compare this code with the [official version](https://github.com/megvii-model/ShuffleNet-Series/tree/master/OneShot) and replace that dataset with CIFAR-10 to have a try.

**Comment by NiuRc** [comment]
> > I replace the dataset with Cifar10 ,and reduce the number of layers.When i training this model ,the accuracy rate maintain 10%.It can't change
> > As far as I am concerned, I think may be the learning rate is to blame which is different from the official version and may be not appropriate. Additionally, the initial state is very important. Maybe you can compare this code with the [official version](https://github.com/megvii-model/ShuffleNet-Series/tree/master/OneShot) and replace that dataset with CIFAR-10 to have a try.

Sorry, have you tried training your model on the imagenet dataset? I ran the CIAFAR10 dataset with the official code, and the loss will be reduced! I am confused and need your help.

**Comment by ShunLu91** [comment]
> > > I replace the dataset with Cifar10 ,and reduce the number of layers.When i training this model ,the accuracy rate maintain 10%.It can't change
> > > As far as I am concerned, I think may be the learning rate is to blame which is different from the official version and may be not appropriate. Additionally, the initial state is very important. Maybe you can compare this code with the [official version](https://github.com/megvii-model/ShuffleNet-Series/tree/master/OneShot) and replace that dataset with CIFAR-10 to have a try.
> 
> Sorry, have you tried training your model on the imagenet dataset? I ran the CIAFAR10 dataset with the official code, and the loss will be reduced! I am confused and need your help.

Maybe you can contact me via lushun19@mails.ucas.ac.cn for detail and we can have a further talk.

**Comment by ShunLu91** [comment]
This issue is caused by the lack of relu and it has been updated.
